app/grants/views.py

```diff
@@ -1006,7 +1006,7 @@ def grants_type_redirect(request, grant_type):
     base_url = reverse('grants:grants_by_category', kwargs={"grant_type":grant_type})
     query_string =  urlencode(request.GET)
     url = '{}?{}'.format(base_url, query_string)
-    return redirect(url)
+    return redirect(url, code=302)
```

Comment:
Github is flagging this as well. If this is a false positive, let's "Dismiss" and mark it as that.